packages/charts/src/vaadin-chart.js

```diff
@@ -1081,6 +1081,12 @@ class Chart extends ResizeMixin(ElementMixin(ThemableMixin(PolymerElement))) {
     if (this.configuration) {
       this.configuration.destroy();
       this.configuration = undefined;
+
+      // Reset series objects to avoid errors while detached
+      const seriesNodes = Array.from(this.childNodes).filter(this.__filterSeriesNodes);
+      seriesNodes.forEach((series) => {
+        series.setSeries(null);
```

Comment:
This does fix the error we had with the Dashboard React example. With the Flow examples though, the chart still remains empty after a detach/attach (widgets reorder):

https://github.com/user-attachments/assets/a54fc57c-01cd-4d94-9f5a-a0a108af1b74

Reply:
In Dashboard's case the issue would get worked around by https://github.com/vaadin/web-components/pull/8115#discussion_r1834317145 however

Reply:
Apparently the Flow component doesn't use `<vaadin-chart-series>` objects but sets the `series` property  (among others) to the `configuration` object programmatically. I think the configuration object never gets restored on reconnect.

State before and after detach/attach:
![Screenshot 2024-11-08 at 14 43 20](https://github.com/user-attachments/assets/789b0858-60e1-4473-a2ec-4ea50e1a5bbe)

Reply:
That sounds like a problem especially since it wasn't caught by ITs, could you please create a separate issue?